live_score_flutter_app/lib/screens/create_game_screen.dart

```diff
@@ -1,65 +1,121 @@
 import 'package:flutter/material.dart';
+import 'package:live_score_flutter_app/providers/games_admin_provider.dart';
 import 'package:live_score_flutter_app/screens/edit_game_screen.dart';
+import 'package:live_score_flutter_app/screens/user_screen.dart';
+import 'package:provider/provider.dart';
 
-class CreateGameScreen extends StatelessWidget {
+class CreateGameScreen extends StatefulWidget {
   static const id = 'creategame';
 
+  @override
+  State<CreateGameScreen> createState() => _CreateGameScreenState();
+}
+
+class _CreateGameScreenState extends State<CreateGameScreen> {
   final team1TextController = TextEditingController();
+
   final team2TextController = TextEditingController();
+
   final descriptionTextController = TextEditingController();
+
   final formKey = GlobalKey<FormState>();
 
+  String selectedItem = 'Football ⚽';
+
+  List<String> gamesList = ['Football ⚽', 'Cricket 🏏', 'Tennis 🎾'];
+
   @override
   Widget build(BuildContext context) {
```

Comment:
instead of writing `MediaQuery.of(context).size.width` every time just define `final size = MediaQuery.of(context).size`

then you can use `width: 0.75* size.width`

Reply:
Noted😊👍